src/env.js

```diff
@@ -29,8 +24,8 @@ export const env = createEnv({
     GOOGLE_CLIENT_SECRET: z.string(),
     GOOGLE_REDIRECT_URI: z.string().url(),
 
-    AWS_ACCESS_KEY_ID: z.string(),
-    AWS_SECRET_ACCESS_KEY: z.string(),
+    AWS_ACCESS_KEY_ID: z.string().optional(),
```

Comment:
could u elaborate on why making this optional?

Reply:
AWS provides temporary credentials within the EC2 instance for accessing AWS resources when an IAM role with appropriate policy is attached to the instance. The AWS SDK automatically retrieves these credentials from the instance, eliminating the need for using long-lived access/secret keys.. [See](https://aws.amazon.com/blogs/security/a-safer-way-to-distribute-aws-credentials-to-ec2/)